docs/manual/guides/dca-adjustments.de.md

```diff
@@ -82,3 +102,59 @@ unset($GLOBALS['TL_DCA']['tl_files']['list']['sorting']['panelLayout']);
 ```
 {{% /expand %}}
 
+
+{{% expand "H-Tag in Überschriften einschränken" %}}
+```php
+// contao/dca/tl_files.php
+$GLOBALS['TL_DCA']['tl_content']['fields']['headline']['options']= ['h2','h3']; # Beispiel auf h2 und h3 einschränken
+```
+{{% /expand %}}
+
+
+{{% expand "Vorbelegungen Playergrösse" %}}
+```php
+// contao/dca/tl_content.php
+$GLOBALS['TL_DCA']['tl_content']['fields']['playerSize']['default'] = [960,540];
+```
+{{% /expand %}}
+
+
+{{% expand "Vorbelegungen Bildeinstellung" %}}
+```php
+// contao/dca/tl_content.php
+// Die Voreinstellungen werden für alle Inhalte mit Bildelementen übernommen. Bild, Galerie
+
+// Vorschaubilder pro Reihe
+$GLOBALS['TL_DCA']['tl_content']['fields']['perRow']['default'] = '4';
+
+// Großansicht/Neues Fenster anhaken
+$GLOBALS['TL_DCA']['tl_content']['fields']['fullsize']['default'] = '1';
+
+// Vorauswahl Bildabstände in px
+$GLOBALS['TL_DCA']['tl_content']['fields']['imagemargin']['default'] = serialize(['unit' => 'px']);
+
+// Sortieren nach Individuelle Reihenfolge
+$GLOBALS['TL_DCA']['tl_content']['fields']['sortBy']['default'] = 'custom'; 
+// Sortieren nach Dateiname (aufsteigend)
+$GLOBALS['TL_DCA']['tl_content']['fields']['sortBy']['default'] = 'name_asc'; 
+// Sortieren nach Dateiname (absteigend)
+$GLOBALS['TL_DCA']['tl_content']['fields']['sortBy']['default'] = 'name_desc'; 
+// Sortieren nach Datum (aufsteigend)
+$GLOBALS['TL_DCA']['tl_content']['fields']['sortBy']['default'] = 'date_asc'; 
+// Sortieren nach Datum (absteigend)
+$GLOBALS['TL_DCA']['tl_content']['fields']['sortBy']['default'] = 'date_desc'; 
+// Sortieren nach Zufällige Reihenfolge
+$GLOBALS['TL_DCA']['tl_content']['fields']['sortBy']['default'] = 'random'; 
+
+// Bildgröße zum Beispiel Exaktes Format Mitte | MItte
+$GLOBALS['TL_DCA']['tl_content']['fields']['size']['default'] = [500,500,'center_center'];
+// weitere Variablen für Exaktes Format:
+// 'crop', 'left_top', 'left_center', 'left_bottom', 'center_top', 'center_bottom', 'right_top', 'right_center', 'right_bottom'
+$GLOBALS['TL_DCA']['tl_content']['fields']['size']['default'] = [150]; # Bildbreite von 150px
+$GLOBALS['TL_DCA']['tl_content']['fields']['size']['default'] = [150,150]; # Bildbreite von 150 x 150px
+
+// Eigene Bildgrössen
+$GLOBALS['TL_DCA']['tl_content']['fields']['size']['default'] = [0, 0, 2]; # die '2' ist die ID der Bildgrösse
+```
```

Comment:
Best practise in Contao is to set the default directly on the database level, instead of via the `default` DCA setting. Though if you want to implement dynamic defaults via an event listener, then you might still want to use the `default` DCA setting.

Reply:
Shall I delete this proposal

Reply:
Let's leave it as it is for now :)